src/classes/issues-processor.ts

```diff
@@ -571,6 +578,10 @@ export class IssuesProcessor {
     if (this._shouldRemoveStaleWhenUpdated(issue) && issueHasComments) {
       await this._removeStaleLabel(issue, staleLabel);
 
+      // Are there labels to remove or add when an issue is no longer stale?
+      await this._removeLabelsWhenUpdatedFromStale(issue, removeLabelsWhenUpdatedFromStale);
```

Comment:
Decided to explicitly keep the `FromStale` as part of the name. In theory, some future update could have automation do stuff on _any_ update. This PR is specifically for issues becoming "un-stale"